azure-pipelines.yml

```diff
@@ -41,6 +41,19 @@ steps:
         zipAfterPublish: false
         modifyOutputPath: false
 
+    - script: |
+        dir /b /s $(Build.ArtifactStagingDirectory)
+        dir /b /s $(System.DefaultWorkingDirectory)
+      displayName: 'List contents of ArtifactStagingDirectory and DefaultWorkingDirectory'
+
+    - task: VSTest@2
+      displayName: 'Run Unit Tests'
```

Comment:
Might be worth it to label it as "Run Sample Unit Tests" since these mostly serve as a demonstration rather than an actual test.